react/components/InstallmentsRenderer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import { FormattedNumber } from 'react-intl'
+import { useCssHandles } from 'vtex.css-handles'
+import { FormattedCurrency } from 'vtex.format-currency'
+import { IOMessageWithMarkers } from 'vtex.native-types'
+
+import { BasicPriceProps } from '../types'
+
+export interface Installments {
+  Value: number
+  InterestRate: number
+  TotalValuePlusInterestRate: number
+  NumberOfInstallments: number
+}
+
+const CSS_HANDLES = [
+  'installments',
+  'installmentsNumber',
+  'installmentValue',
+  'installmentsTotalValue',
+  'interestRate',
+] as const
+
+interface Props extends BasicPriceProps {
+  installments: Installments
+}
+
+function InstallmentsRenderer(props: Props) {
```

Comment:
I'm not quite sure about this component name, open to suggestions

Reply:
`InstallmentsComponent` is too redundant I think and `InstallmentsFragment` seems to mean something else